@here/harp-mapview/lib/Utils.ts

```diff
@@ -26,29 +26,30 @@ const groundNormalPlanarProj = new THREE.Vector3(0, 0, 1);
 const groundNormalInvPlanarProj = groundNormalPlanarProj.clone().negate();
 const groundPlane = new THREE.Plane(groundNormalPlanarProj.clone());
 const groundSphere = new THREE.Sphere(undefined, EarthConstants.EQUATORIAL_RADIUS);
-const cameraZPosition = new THREE.Vector3(0, 0, 0);
-const rotationMatrix = new THREE.Matrix4();
-const unprojectionMatrix = new THREE.Matrix4();
 const rayCaster = new THREE.Raycaster();
-const yawQuaternion = new THREE.Quaternion();
-const pitchQuaternion = new THREE.Quaternion();
-const tmpQuaternion = new THREE.Quaternion();
-const tmpMatrix = new THREE.Matrix4();
 const tmpVectors: THREE.Vector3[] = [new THREE.Vector3(), new THREE.Vector3(), new THREE.Vector3()];
 
-export namespace MapViewUtils {
-    /**
-     * The yaw axis around we rotate when we change the yaw.
-     * This axis is fix and is the -Z axis `(0,0,1)`.
-     */
-    const yawAxis = new THREE.Vector3(0, 0, 1);
-
-    /**
-     * The pitch axis which we use to rotate around when we change the pitch.
-     * The axis is fix and is the +X axis `(1,0,0)`.
-     */
-    const pitchAxis = new THREE.Vector3(1, 0, 0);
+/**
+ * Cached ThreeJS instances for realtime maths.
+ */
+const space = {
```

Comment:
What does this actually describes/stores? It's really tricky to just say by its name...

Reply:
`space` and `tangentSpace` below are used to do some space transformation math when a tangent space is involved, computed through a new method `localTangentSpace` implemented a bit earlier in another PR. Unfortunately the module it belongs to cannot depend on ThreeJS yet, so it has to implement `Vector3Like` for the math used in it. We were not keen on having a `Matrix3Like`, since legal might agree on the ThreeJS dependency soon. It might not be necessary to explain these internal and momentary legal issues there, and let people with a definition of a space as 3 axes, in the `localTangentSpace` method. This is also straightforward to use in the associated code, I believe?